.github/workflows/score_new_plugins.yml

```
@@ -0,0 +1,111 @@
+name: Trigger scoring run
+
+
+# Triggered on all PRs on merge to main
+# If changes are made to a subdir of /benchmarks or /models,
+# a Jenkins scoring run is triggered for the corresponding plugin
+
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+
+permissions: write-all
+
+jobs:
+  changes_models_or_benchmarks:
+    name: Check if PR makes changes to /models or /benchmarks
+    runs-on: ubuntu-latest
+    outputs:
+      PLUGIN_INFO: ${{ steps.getpluginfo.outputs.PLUGIN_INFO }}
+      RUN_SCORE: ${{ steps.runscore.outputs.RUN_SCORE }}
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Get changed files
+        uses: dorny/paths-filter@v2.11.1
+        id: filter
+        with:
+          list-files: shell
+          filters: |
+            changed:
+              - '**'
+      - name: Save changed files to env var
+        run: echo "CHANGED_FILES=${{ steps.filter.outputs.changed_files }}" >> $GITHUB_ENV
+      
+      - name: Parse changed files with python script
+        run: |
+          echo "PLUGIN_INFO=$(python .github/workflows/parse_changed_files.py '${{ env.CHANGED_FILES }}')" >> $GITHUB_ENV
+          
+      - name: Save plugin info to outputs
+        id: getpluginfo
+        run: |
+          echo "PLUGIN_INFO=$PLUGIN_INFO" >> $GITHUB_OUTPUT
+
+      - name: check if scoring needed
+        id: runscore
+        run: |
+          echo "RUN_SCORE=$(jq -r '.run_score' <<< "$PLUGIN_INFO")" >> $GITHUB_OUTPUT
+
+  get_submitter_info:
+    name: Get PR author email and (IF web submission) Brain-Score user ID
```

```suggestion
    name: Get PR author email and (if web submission) Brain-Score user ID
```
thought IF was an abbreviation at first